packages/react-app/src/helpers/EIP1559Helper.js

```diff
@@ -41,7 +41,7 @@ export const createEthersWallet = ethersProvider => {
 
 // Create EIP-1559 type-2 transactions on mainnet and polygon
 // Other networks use legacy transactions with gasPrice
-export const sendTransaction = (txParams, signer, injectedProvider) => {
+export const sendTransaction = (txParams, signer, injectedProvider, relayerWallet) => {
```

Comment:
Maybe you can change the params to be a hash, so then you don't have to send something like ```sendTransaction(txParams, undefined, undefined, relayerWallet)```